BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -657,9 +673,7 @@ namespace big
 								secondary_color = color;
 							}
 
-							g_fiber_pool->queue_job([] {
-								VEHICLE::SET_VEHICLE_COLOURS(player_vehicle, primary_color, secondary_color);
-							});
+							VEHICLE::SET_VEHICLE_COLOURS(player_vehicle, primary_color, secondary_color);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.